### ISSUE TITLE
"[Errno 18] Invalid cross-device link" when upgrade to 2.0

### ISSUE THREAD

**Issue by Tomasy** [Kozea/Radicale#635]
When I try to upgrade to 2.0 and execute I got
```
INFO: Exporting storage for Radicale 2.0.0 to '/var/lib/radicale2'
ERROR: Can't create '/var/lib/radicale2' directory: [Errno 18] Invalid cross-device link
```
I have used radicale on a Fedora 24 host with version 1.1.2 To upgrade it I downloaded radicale version 1.1.3 and executed it with
`PYTHONPATH=/root/tmp/Radicale-1.1.3 ./bin/radicale --export-storage=/var/lib/radicale2`
and recived the error message above

**Comment by Unrud** [comment]
That's an unexpected error code. Radicale creates the new storage in a temporary folder in ``/tmp/Radicale.export....`` and then moves/copies the result with [shutil.move](https://docs.python.org/3.6/library/shutil.html#shutil.move) to the target folder. Try to export into a tempoarary folder like ``/tmp/radicale2`` and move the result by hand to ``/var/lib/radicale2``.

**Comment by Tomasy** [comment]
That solve the problem. In Fedora /tmp is mounted as a tmpfs.
Thanks.

**Comment by Unrud** [comment]
> That solve the problem. In Fedora /tmp is mounted as a tmpfs.

The documentation of [shutil.move](https://docs.python.org/3.6/library/shutil.html#shutil.move) says, that it works across file systems. I tested it on Fedora 25 and it works fine.

**Comment by Thor77** [comment]
Can reproduce on ArchLinux when exporting to `/var/lib/radicale2`.

**Comment by Unrud** [comment]
Fixed in version 1.1.4. Version 1.1.3 didn't use ``shutil.move``. The change was in the 1.1.x branch, but didn't make it into the release. Sorry for the confusion.
